app/code/Magento/Backend/Api/Search/ItemsInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Backend\Api\Search;
+
+/**
+ * @api
+ */
+interface ItemsInterface
+{
+    const LIMIT = 'limit';
+    const START = 'start';
+    const QUERY = 'query';
+
+    /**
+     * get the search result items
+     *
+     * @return array
+     */
+    public function getResults();
+
+    /**
+     * set offset
+     *
+     * @param int $start
+     * @return ItemsInterface
```

Comment:
we don't recommend to use "fluent interfaces" and "method chaining", we support CQS where the command should return void instead of $this.
Returning $this from the setter method is allowed just for Builder implementation
(I see that we have this chaining in code before your changes, just think that if we started to improve the code quality, let's fix this out)